modules/ROOT/pages/getting-started.adoc

```diff
@@ -49,10 +47,12 @@ Because this project is a Maven project, it contains a `pom.xml` file that conta
 <parent>
   <groupId>org.mule.extensions</groupId>
   <artifactId>mule-modules-parent</artifactId>
-  <version>1.6.3</version>
+  <version>1.9.0</version>
 </parent>
 ----
 
+NOTE: The parent POM defines the minimum Mule version, which must be compatible with the target Mule version.
```

Comment:
This is not sufficient to address the original comment. This is the getting started page. At this point, the reader has no context to understand what any of this means. They also have no way of knowing which parent pom version corresponds to which mule version nor the supported byte code level